src/System.Drawing.Common/src/System/Drawing/SystemIcons.cs

```diff
@@ -41,30 +42,12 @@ public static Icon Shield
         {
             get
             {
-                Contract.Ensures(Contract.Result<Icon>() != null);
-
-                if (s_shield == null)
-                {
-                    try
-                    {
-                        // we hard-code size here, to prevent breaking change
-                        // the size of _shield before this change is always 32 * 32  
-                        IntPtr hIcon = IntPtr.Zero;
-                        int result = SafeNativeMethods.LoadIconWithScaleDown(NativeMethods.NullHandleRef, SafeNativeMethods.IDI_SHIELD, 32, 32, ref hIcon);
-
-                        if (result == 0)
-                            s_shield = new Icon(hIcon);
-                    }
-                    catch
-                    {
-                        // we don't want to throw exception here.
-                        // If there is an exception, we will load an icon from file ShieldIcon.ico
-                    }
-                }
```

Comment:
From the commit:

> This API always throws an EntryPointNotFoundException since some version
since either Windows 7 or Windows 8
I'm willing to revert this if the API is present in Windows 7.

This always throws an EntryPointNotFoundException as the API doesn't exist anymore. If it doesn't exist on Windows 7, it can be removed.